USAGEGUIDE.md

```diff
@@ -125,117 +72,56 @@ make deploy
 make undeploy
 ```
 
-## Kubernetes Operator Deployment with Service Account
-
-#### Create Kubernetes Secret for OP_SERVICE_ACCOUNT_TOKEN ####
-
-Create a Service Account token for the operator and save it as a Kubernetes Secret:
-
-```bash
-kubectl create secret generic onepassword-service-account-token --from-literal=token="$OP_SERVICE_ACCOUNT_TOKEN"
-```
-
-If you do not have a token for the operator, you can generate a token and save it to Kubernetes with the following command:
-
-```bash
-kubectl create secret generic onepassword-service-account-token --from-literal=token=$(op service-account create my-service-account --vault Dev:read_items --vault Test:read_items,write_items)
-```
+---
 
-**Deploying the Operator**
+## Use Kubernetes Operator with Connect
 
-An sample Deployment yaml can be found at `/config/manager/manager.yaml`.
-To use Operator with Service Account, you need to set the `OP_SERVICE_ACCOUNT_TOKEN` environment variable in the `/config/manager/manager.yaml`. And remove `OP_CONNECT_TOKEN` and `OP_CONNECT_HOST` environment variables.
+### 1. [Deploy with Helm](https://developer.1password.com/docs/k8s/operator/?deployment-type=helm#helm-step-1)
+### 2. [Deploy manually](https://developer.1password.com/docs/k8s/operator/?deployment-type=manual#manual-step-1)
```

Comment:
`[nit]`
I can understand relying on the Developer Docs as the source of truth, but I feel like this experience of back and forth will not make onboarding better. :confused:

Reply:
I see it only one direction. Developer Docs are the source of truth.

What are you concerns? What are you ideas to make the onboarding better?

Reply:
Yes, this will be mentioned in dev portal.